selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/miscellaneous/NavigateToFileTest.java

```diff
@@ -67,6 +71,7 @@
       Arrays.asList(
           "classpath (/NavigateFile_2/.che)", "classpath (/NavigateFile/.che)",
           "createdFrom.con (/NavigateFile_2)", "createdFrom.api (/NavigateFile)");
+  private static final String FILE_H_SYMBOL = "HEAD (/NavigateFile/.git)";
```

Comment:
Could you, please, add comment or better to rename the constant to explain the exact proposal of it?